baselines/asset/samples/generated/v1/asset_service.batch_get_assets_history.js.baseline

```diff
@@ -0,0 +1,68 @@
+
+
+
+'use strict';
+
+function main() {
+  // [START asset_batch_get_assets_history_sample]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  /**
+   *  Required. The relative name of the root asset. It can only be an
+   *  organization number (such as "organizations/123"), a project ID (such as
+   *  "projects/my-project-id")", or a project number (such as "projects/12345").
+   */
+  // const parent ='abc123'
+  /**
+   *  A list of the full names of the assets. For example:
+   *  `//compute.googleapis.com/projects/my_project_123/zones/zone1/instances/instance1`.
+   *  See [Resource
+   *  Names](https://cloud.google.com/apis/design/resource_names#full_resource_name)
+   *  and [Resource Name
+   *  Format](https://cloud.google.com/asset-inventory/docs/resource-name-format)
+   *  for more info.
+   *  The request becomes a no-op if the asset name list is empty, and the max
+   *  size of the asset name list is 100 in one request.
+   */
+  // const assetNames ='abc123'
+  /**
+   *  Optional. The content type.
+   */
+  // const [contentType] =''
+  /**
+   *  Optional. The time window for the asset history. Both start_time and
+   *  end_time are optional and if set, it must be after the current time minus
+   *  35 days. If end_time is not set, it is default to current timestamp.
+   *  If start_time is not set, the snapshot of the assets at end_time will be
+   *  returned. The returned results contain all temporal assets whose time
+   *  window overlap with read_time_window.
+   */
+  // const [readTimeWindow] =''
+
+  // Imports the Asset library
+  const {AssetServiceClient} = require('@google-cloud/asset');
+
+  // Instantiates a client
+  const assetClient = new AssetServiceClient();
+
+  async function batchGetAssetsHistory() {
+    // Construct request
+    const request = {
+      parent,
+    };
+
+    // Run request
+    const response = await assetClient.batchGetAssetsHistory(request);
+    console.log(`Response: ${response}`);
```

Comment:
In the majority of cases, I think this will result in a `[obj Object]` right?  Should we consider a `util.inspect`, or is that too far?  Curious what @bcoe thinks here too.

Reply:
So, if this is an async list function, we will iterate through the results. Other than that though, I'm not sure how we'd distinguish what to iterate through.

Just out of curiosity, why wouldn't `JSON.stringify` work?

Reply:
I think we should just `console.log(response)`, which will output it without `[obj Object]` but doesn't require `JSON.stringify`.

I think for `util.inspect` we'd need ti `require('util')`.